pkgs/development/python-modules/deepdiff/default.nix

```diff
@@ -56,6 +67,9 @@ buildPythonPackage rec {
     pytestCheckHook
     python-dateutil
     tomli-w
+    polars
+    pandas
+    pytest-benchmark
```

Comment:
> We don't want to instrument benchmarks in check phase. It will only slow down the build server.

I too wanted to avoid that, but I don't know how to make the build phase pass without this :/.

Reply:
> > We don't want to instrument benchmarks in check phase. It will only slow down the build server.
> 
> I too wanted to avoid that, but I don't know how to make the build phase pass without this :/.

Same here - disabled the relevant tests.